manifests/server/config.pp

```diff
@@ -194,19 +194,23 @@
     }
   }
 
-  if $facts['os']['family'] == 'RedHat' {
-    if $facts['os']['release']['major'] == '8' or ($facts['os']['name'] == 'Fedora' and $facts['os']['release']['major'] >= '32') {
+  # RHEL 7 and 8 both support drop-in files for systemd units.  The old include directive is deprecated and may be removed in future systemd releases.
+  # Gentoo also supports drop-in files.
+  if ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] =~ /7|8/) or ($facts['os']['name'] == 'Fedora' and $facts['os']['release']['major'] >= '32') or $facts['os']['family'] == 'Gentoo' {
```

Comment:
I'd also use drop ins on Fedora and simplify it to:
```suggestion
  if $facts['os']['family'] in ['RedHat', 'Gentoo'] and $facts['service_provider'] == 'systemd' {
```

This uses the [service_provider](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/lib/facter/service_provider.rb) fact from stdlib to detect the service provider.